projects/RabbitMQ.Client/client/impl/Connection.cs

```diff
@@ -213,8 +213,12 @@ internal void TakeOver(Connection other)
 
         internal IConnection Open()
         {
-            return OpenAsync()
-                .ConfigureAwait(false).GetAwaiter().GetResult();
+            /*
+             * https://devblogs.microsoft.com/dotnet/configureawait-faq/
+             * I'm using GetAwaiter().GetResult(). Do I need to use ConfigureAwait(false)?
+             * Answer: No
+             */
+            return OpenAsync().GetAwaiter().GetResult();
```

Comment:
I find the approach the Azure SDK uses more intention revealing

https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/src/Shared/TaskExtensions.cs#L27

I would recommend using a similar TaskExtension if you continue doing sync over async

Reply:
Version 7 of this client is the last version to support a sync API but we'll probably be supporting it for several years. I'll copy an extension for this.

Reply:
Done!